### ISSUE TITLE
Replace Reactrap in favor of Chakra UI

### ISSUE THREAD

**Issue by samuelcastro** [UnlyEd/next-right-now#344]
[Chakra UI](https://chakra-ui.com/) is a very interesting react UI library that's growing up in popularity by its simplicity, [design practices](https://chakra-ui.com/docs/principles) and accessibility. Also it will eventually support [zero-runtime CSS-in-JS](https://github.com/chakra-ui/chakra-ui/issues/4003) capabilities. 

I think adopting Chakra would improve the dev experience, accessibility and performance in the future.

Thoughts?

**Comment by Vadorequest** [comment]
I completely agree. We've ditched bootstrap/reactstrap from our 2 last apps built with NRN and it's amuch better experience overall.

The main reason why I haven't updated NRN is... it takes time. 😅
